.github/workflows/pypi.yaml

```diff
@@ -0,0 +1,17 @@
+name: PyPI
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  PyPIPoetryPublish:
+    name: PyPI Poetry Publish
+    uses: uc-cdis/.github/.github/workflows/python_package_index_publish.yaml@master
+    with:
+        PYTHON_VERSION: '3.9'
+        # the package was registered under someone's personal test-pypi account, so we
+        # can't push there
```

Comment:
is this true for this repo? if so can we move it out to gen3?

Reply:
Thanks for the catch. I need to check on that. I got the pypi.yaml file from gen3utils and like a computer, my mind completely ignored the comments 😅

Reply:
It appears there's no project named `gen3Config` on Test PyPI ([link here](https://test.pypi.org/search/?q=gen3config)). I believe we'll need to create one for the `uc-ctds-test` user.

Reply:
It'll be created automatically, we just need to remove this comment from here